astro/src/content/docs/get-started/start-here/step-4.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Step 4 - Role-Based Access Control"
+description: Learn how to use FusionAuth by using an example application that has a protected page defined.
+section: get started
+subcategory: start here
+navOrder: 0
+codeRoot: https://raw.githubusercontent.com/fusionauth/fusionauth-example-get-started/main
```

Comment:
I don't think you need `codeRoot` for any of these docs.

Reply:
Unless I use RemoteCode, right? 😉